kotlinpoet-km-specs/src/main/kotlin/com/squareup/kotlinpoet/km/specs/KmSpecs.kt

```diff
@@ -681,6 +739,25 @@ private fun ImmutableKmWithFlags.addVisibility(body: (KModifier) -> Unit) {
   }
 }
 
+// TODO This is a copy of the stdlib version. Use it directly once it's out of experimental
```

Comment:
Maybe mark with `KotlinPoetKm`?

Reply:
No need to, it's private and doesn't use any metadata APIs